playground/nextjs/.env.example

```
@@ -8,3 +8,10 @@ OPENAI_API_KEY=
 NOVU_SECRET_KEY=
 NOVU_SUBSCRIBER_ID=
 NOVU_HITL_WORKFLOW_ID=refund-approval
+
+# Slack Connect Chat demo
+NEXT_PUBLIC_NOVU_SLACK_INTEGRATION_IDENTIFIER=
+NEXT_PUBLIC_SLACK_USER_ID=
+SLACK_BOT_USER_OAUTH_TOKEN=
+# Workflow ID to trigger when clicking "Send Test Message" on the connect-chat playground page
+NEXT_PUBLIC_NOVU_SLACK_TEST_WORKFLOW_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_NOVU_SLACK_TEST_WORKFLOW_ID key should go before the NEXT_PUBLIC_SLACK_USER_ID key

(UnorderedKey)